index.bs

```diff
@@ -5044,14 +5044,15 @@ Liaisons</h2>
 	confidentiality agreements;
 	and mutual membership agreements.
 
-	The [=CEO=] <em class="rfc2119">may</em> negotiate
+	W3C <em class="rfc2119">may</em> negotiate
 	a Memorandum of Understanding with another organization.
 	For purposes of the W3C Process, a <dfn id="mou" export lt="Memorandum of Understanding | Memoranda of Understanding | MoU">Memorandum of Understanding</dfn> (<abbr>MoU</abbr>)
-	is a formal agreement or similar contractual framework between W3C and another party or parties,
-	other than between W3C and W3C members
+	is a formal agreement or similar contractual framework between W3C and another party or parties--
+	other than between W3C and W3C Members
 	for the purposes of membership,
+	between W3C and its Partners [[BYLAWS]],
 	and other agreements related to the ordinary provision of services
-	for the purpose of running W3C,
+	for the purpose of running W3C--
```

Comment:
em-dashes are generally preferred to double-hyphens.  I find em-dashes surrounded with spaces to be more readable, but guidance on this varies, so I'll try to conform to Editor's preference.
```suggestion
	for the purpose of running W3C —
```

Reply:
The bikeshed spec preprocessor we use turns double hyphens at the end of the line into em-dashes when it renders the document, so we should be good already.